### PR TITLE
Update Marketplace Contribution Documentation

### PR DESCRIPTION
**Pull request by avidan-H**
## Status
Ready

## Screenshots
Step 3 now has tabs - one describes the form fill for `Creating a New Pack` and the other tab describes filling the form for `Updating an Existing Pack`
image
image

## Description
Updates the Marketplace contribution documentation to include information about updating existing content packs from the contribution flow in the UI.

### CODE REVIEW

File: docs/contributing/marketplace.md [hunk 1]
```diff
@@ -11,19 +11,67 @@ This article walks you through the process of contributing content from the Cort
...
 8. You can now modify the files changed in the pull request as part of the review process.
 
+
 :::note
 In order to update the content pack that you contributed, you will need to modify the files in the pull request created in GitHub or close the pull request and create a new contribution.
 :::
+
+:::note Updating Existing Content
+<p><a id="footnote1"><em>1</em></a>: The contribution mode selection dropdown will only appear if content items that were part of your contribution were detected as originating from existing sources (for example, you created a new automation in the UI by clicking "Duplicate Automation").</p>
+<p><a id="footnote2"><em>2</em></a>: The pack options when selecting an existing pack to update will be populated by the packs from which the content items of which your content was duplicated from, originally belong to.</p>
```
Comment (avidan-H) [inline docs/contributing/marketplace.md:75]: @Itay4 the language here is _really_ bad and needs improving... I can't think how to succinctly describe this.

**Comment by Itay4** [comment]
@avidan-H the spacing here looks weird
image

**Comment by avidan-H** [comment]
@Itay4 yea that's one of the weird formatting things that happens when you press the anchor links and I'm not sure why.

**Review by glicht — COMMENTED** [review]
I don't really like the tabs concept. I think it is confusing. Tabs are usually not used like this.

The double note call out at the bottom of the page is too much. Let's just create a `Notes` section and put there all notes in bullets.

**Comment by avidan-H** [comment]
@kirbles19 @ShirleyDenkberg is one of you able to review this updated documentation?
* [edit textual contents](https://github.com/demisto/content-docs/edit/document-update-existing-flow/docs/contributing/marketplace.md?pr=%2Fdemisto%2Fcontent-docs%2Fpull%2F364)
* [docs preview site](https://pull-request-364--demisto-content-docs.netlify.app/)

**Comment by kirbles19** [comment]
@avidan-H Done.

**Review by Itay4 — APPROVED** [review]

**Pull request merged**